js/activity.js

```diff
@@ -1988,7 +1988,7 @@ function Activity() {
                     stage.update();
                     break;
                 case KEYCODE_DOWN:
-                    textMsg('DOWN ARROW ' + _('Moving block down.'));
+                    textMsg('UP ARROW ' + _('Moving block down.'));
```

Comment:
Please rebase before making your PR. This is undoing an unrelated commit and is wrong.